lib/infrastructure/data_access_layer/repository/algorand_service.dart

```diff
@@ -99,8 +99,8 @@ class AlgorandService {
 
     if (amount.assetId == 0) {
       final payTxn = await (PaymentTransactionBuilder()
-            ..sender = Address.fromAlgorandAddress(address)
-            ..receiver = Address.fromAlgorandAddress(SYSTEM_ACCOUNT[net]!)
+            ..sender = Address.fromAlgorandAddress(address: address)
```

Comment:
are you changing algorand version down again?

Reply:
I think its old accept yours **algorand_dart: ^2.0.0-dev.4**